doc/source/whatsnew/v0.21.0.txt

```diff
@@ -43,9 +43,10 @@ Backwards incompatible API changes
 
 - Support has been dropped for Python 3.4 (:issue:`15251`)
 - The Categorical constructor no longer accepts a scalar for the ``categories`` keyword. (:issue:`16022`)
-
 - Accessing a non-existent attribute on a closed :class:`HDFStore` will now
   raise an ``AttributeError`` rather than a ``ClosedFileError`` (:issue:`16301`)
+- Compression defaults in HDF stores now follow pytable standards. Default type is 'zlib' and default level is 0 (:issue:`15943`)
```

Comment:
@linebp `complevel=0` means no compression right? i.e. `complevel=0, complib='zlib'` would be byte-for-byte identical to `complevel=0, complib='lzo'`, since it's not actually compressed?